### ISSUE TITLE
Refresh lint tasks

### ISSUE THREAD

**Issue by GaryJones** [craigsimps/gulp-wp-toolkit#60]
The current lint tasks are:

```js
    'lint:css': [require('./lint/stylelint')],
    'lint:scss': [require('./lint/scss')],
    'lint:eslint': [require('./lint/eslint')],
    'lint:jshint': [require('./lint/jshint')],
    'lint:jscs': [require('./lint/jscs')],
    'lint:jsvalidate': [require('./lint/jsvalidate')],
    'lint:json': [require('./lint/json')],
    'lint:js': [sequence('lint:jshint', 'lint:jscs', 'lint:jsvalidate', 'lint:json')],
    'lint:i18n': [require('./lint/i18n')],
    'lint:colors': [require('./lint/colors')],
    'lint:phpcs': [require('./lint/phpcs')],
    'lint:phpmd': [require('./lint/phpmd')],
    'lint:php': [sequence('lint:phpcs', 'lint:phpmd')],
    'lint': [sequence('lint:php', 'lint:scss', 'lint:js', 'lint:i18n', 'lint:colors')],
```

I reckon we could re-jig some of those:

- include `eslint` in the `js` task, possibly drop one or two which eslint covers.
- create a `lint:style` that can handle `css`, `scss`, and `colors`,
- Add `i18n` into `php` task.

cc @ntwb who is more familiar with what eslint and stylelint can check for here.

Some of these individual tasks also seem broken (not finding the right config files, so this also needs fixing up.

**Comment by ntwb** [comment]
ESLint will cover what JSHint and JSCS have previously so dropping those 2 for JS
stylelint will cover both CSS and SCSS, scss-lint can then be dropped

**Comment by GaryJones** [comment]
@ntwb Thanks.

Whilst you're here, are we using [eslint](https://github.com/craigsimps/gulp-wp-toolkit/blob/cb52ccebd663dd4859275938541ad6f545b250f5/tasks/lint/eslint.js) and [stylelint](https://github.com/craigsimps/gulp-wp-toolkit/blob/cb52ccebd663dd4859275938541ad6f545b250f5/tasks/lint/stylelint.js) in the best way possible?

**Comment by GaryJones** [comment]
Well, it was a nice idea, but now none of the tasks are working. Debugging.

**Comment by ntwb** [comment]
> Whilst you're here, are we using eslint and stylelint in the best way possible?

It looks like it, one issue I've fixed in #64
